docs/guide/dev/deep_dive/advanced_content_filter/README.md

```diff
@@ -112,6 +112,24 @@ In browsers where it is possible to recognize whether the content comes (was cop
 
 Read more about the paste filter in the {@linkapi CKEDITOR.config#pasteFilter } documentation.
 
+## Supplying Paste Tools
+
+Plugins derived from [Paste Tools](https://ckeditor.com/cke4/addon/pastetools) family, like [Paste From Word](https://ckeditor.com/cke4/addon/pastefromword) and [Paste From Google Docs](https://ckeditor.com/cke4/addon/pastefromgdocs) ensures their filtering features used to clean up content provided by 3rd party apps. Nevertheless, they are mostly the first filter against incorrect, nonsemantic HTML packed into browser clipboard. These plugins will fix HTML structure and clean up meaningless code, but at the same time try to preserve as much information as possible which can be correctly consumed later. A significant part of the content preparation goes to a huge CKEditor 4 plugins ecosystem with predefined ACF rules used to semantically improve HTML content.
+
+As an example, when pasting an image from MS Word document, you should enable one of the image plugins like [Image](https://ckeditor.com/cke4/addon/image) or [Enchanced Image](https://ckeditor.com/cke4/addon/image2), otherwise, the image will be removed from content by default ACF mode.
+
+However, as you probably already read, you can still enhance this behavior by your custom ACF rules and explicitly tell that image should be still preserved despite missing image plugins:
+
+	config.extraAllowedContent = 'img';
+
+### Remove font styles example
+
+One of the features which have been deprecated lately in favor of ACF is {@linkapi CKEDITOR.config#pasteTools_removeFontStyles}, so let's see how the same result could be accomplished with ACF:
```

Comment:
```suggestion
One of the features which have been deprecated recently in favor of ACF is {@linkapi CKEDITOR.config#pasteTools_removeFontStyles}, so let's see how the same result could be accomplished with ACF:
```

Have to check the exact difference between "lately" and "recently", and "recently" seems to a better fit here (see https://english.stackexchange.com/a/32575).

Reply:
Sounds better :+1: